composer.json

```diff
@@ -13,14 +13,14 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "illuminate/config": ">=5.5 <6.0",
-        "illuminate/support": ">=5.5 <6.0",
-        "illuminate/database": ">=5.5 <6.0"
+        "illuminate/config": ">=5.5",
```

Comment:
It would be better to have `^5.5|^6.0` for all `illuminate` packages.
That will include following releases: `5.5.*`, `5.6.*`, `5.7.*`, `5.8.*`, `6.*`.

Reply:
Ok, sure.